code/modules/antagonists/voidwalker/voidwalker.dm

```diff
@@ -42,20 +42,18 @@
 /datum/outfit/voidwalker/post_equip(mob/living/carbon/human/human, visualsOnly)
 	human.set_species(/datum/species/voidwalker)
 
-/datum/objective/voidwalker_fluff
+/datum/objective/voidwalker_objective
 
-/datum/objective/voidwalker_fluff/New()
+/datum/objective/voidwalker_objective/New()
 	var/list/explanation_texts = list(
-		"Show them the beauty of the void.",
-		"They must see what you have seen. They must walk where you have walked.",
-		"Recover what you have lost.",
-		"Obliterate the tyranny of matter.",
-		"Make them all just like you."
+		"Show them the beauty of the void. Drag them into the cosmic abyss, then impart the truth of the void unto them. Seek to enlighten, not destroy.",
+		"They must see what you have seen. They must walk where you have walked. Bring them to the void and show them the truth. The dead cannot know what you know.",
+		"Recover what you have lost. Bring your children into the inky black return them to your flock.",
```

Comment:
```suggestion
		"Recover what you have lost. Bring your children into the inky black and return them to your flock.",
```

Reply:
idk why @Time-Green marked this as resolved, it's clearly not correct as it is. I'll fix it.